recipes/libmaxminddb/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os.path
+
+from conans import ConanFile, CMake, tools
+
+
+class libmaxminddbTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "pkg_config"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            self.run(os.path.join(".", "example"))
```

Comment:
```suggestion
            self.run(os.path.join(".", "example"), run_environment=True)
```

This will allow using shared lib.

Reply:
OK. That makes sense. Included.